runtime/centrifuge/src/lib.rs

```diff
@@ -1180,6 +1277,36 @@ impl pallet_collator_selection::Config for Runtime {
 	type WeightInfo = weights::pallet_collator_selection::WeightInfo<Self>;
 }
 
+/// Xcm Weigher shared between multiple Xcm-related configs.
+pub type XcmWeigher = xcm_builder::FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
+
+parameter_types! {
+	// 1 DOT, which has 10 decimals, should be enough to cover for fees opening/accepting hrmp channels.
+	pub MaxHrmpRelayFee: MultiAsset = (MultiLocation::parent(), 10_000_000_000u128).into();
```

Comment:
Updated to 1 DOT, please let me know if we want to change this.

Reply:
I think it is fine.

Reply:
@cdamian yeah this is fine and we don't plan to use this feature of `xcm-transactor` for now anywyays 👍